tests/lb_dictutil_test.py

```diff
@@ -0,0 +1,225 @@
+"""Tests for Ladybug Geometry dict->Object converter.
+
+Note: Written in PyTest format.
+"""
+
+from ladybug_geometry.geometry2d import (Vector2D, Point2D, Ray2D, 
+                            LineSegment2D, Arc2D, Polyline2D, Polygon2D, Mesh2D)
+from ladybug_geometry.geometry3d import (Vector3D, Point3D, Ray3D,
+                            LineSegment3D, Arc3D, Polyline3D, Polyface3D, Mesh3D,
+                            Plane, Face3D, Sphere, Cone, Cylinder)
+from lb_dictutil import lb_geom_dict_to_object
```

Comment:
Thank you, @PH-Tools ! I will review shortly.  In the meantime, it seems that your tests are not passing because you did not put your module inside of the python package. Could you move it into the `ladybug_geometry` folder and rename it as `dictutil.py`? I would also recommend renaming the function to `geometry_dict_to_object`.

Then you will be able to import your method here using:
```
from ladybug_geometry.dictutil import geometry_dict_to_object
```

Reply:
Hi @chriswmackey  - whoops! my mistake. - thanks for the comments. I'll revise and submit a new PR with those edits and revisions. I will double check that all tests are passing this time before submitting.